lib/rules/no-trailing-spaces.js

```diff
@@ -63,7 +64,10 @@ module.exports = function(context) {
                 linebreaks = context.getSource().match(/\r\n|\r|\n|\u2028|\u2029/g),
```

Comment:
Should we replace deprecated APIs with new APIs in this rule?

- `context.getSource()` → `sourceCode.text`
- `src.split(/\r?\n/)` → `sourceCode.lines`